artifacts.go

```diff
@@ -523,8 +523,13 @@ func run() error {
 	// write
 	//
 	writeRootfs := cli.Command{
-		Name:   "rootfs-image",
-		Action: writeArtifact,
+		Name: "rootfs-image",
+		Action: func(c *cli.Context) error {
+			if len(strings.Fields(c.String("artifact-name"))) > 1 { // check for whitespace in artifact-name
+				return cli.NewExitError("whitespace is not allowed in the artifact-name", 1)
+			}
+			return writeArtifact(c)
+		},
```

Comment:
@kacf took a slightly different approach when rewriting this, and decided to wrap the writeartifact function. Not super clean, but acceptable?

Reply:
No, I think this is quite clean actually, as long as it's as small as that!